### PR TITLE
Updating french language :tada:

### PR DESCRIPTION
**Pull request by thomasbnt**

### CODE REVIEW

**Comment by elamperti** [comment]
Hi! Thanks for your updates! May you join the lang fixes from #69 in this PR? There are some inconsistencies in that other PR (it re-adds the **fr** language in both `index.html` and `i18n.js`)

**Comment by thomasbnt** [comment]
it's already done !

**Comment by elamperti** [comment]
The string for `support.title` ([this one](https://github.com/elamperti/OpenWebScrobbler/pull/69/files#diff-f4e98e22b6447cfa27263bf7dba713a0R12)) in `home.json` was missing in this PR, I added it in another commit. Thanks for your contribution! 🎉

File: public/locales/fr/common.json [hunk 1]
```diff
@@ -9,19 +9,19 @@
...
-  "backToSearch": "Back to search",
-  "userNotFound": "User not found",
-  "invalidUsername": "Invalid username",
-  "recentlySearchedUsers": "Searched recently",
+  "findFriendCopy": "Entrez un nom d'utilisateur last.fm pour voir leurs dernières pistes",
+  "visitProfile": "Voir le profil",
+  "search": "Rechercher",
+  "backToSearch": "Retourner sur la recherche",
+  "userNotFound": "Aucun utilisateur trouvé",
+  "invalidUsername": "Nom d'utilisation invalide",
```
Comment (elamperti) [inline public/locales/fr/common.json]: Shouldn't this be "Nom d'utilisateur invalide"?

```suggestion
  "invalidUsername": "Nom d'utilisateur invalide",
```

Comment (thomasbnt) [reply public/locales/fr/common.json]: Nom d'utilisateur invalide = Invalid username, not 'utilisation' which translates into `use` or also `usage`.

File: public/locales/fr/common.json [hunk 2]
```diff
@@ -9,19 +9,19 @@
...
+  "recentlySearchedUsers": "Récement recherche",
```
Comment (elamperti) [inline public/locales/fr/common.json]: This is for recently searched (past tense) users, is this conjugation correct?

```suggestion
  "recentlySearchedUsers": "Récemment recherchés",
```

File: public/locales/fr/common.json (hunk 1 above)
Comment (elamperti) [reply public/locales/fr/common.json]: ok, it should be `utilisateur` then! 👍

File: public/locales/fr/common.json (hunk 2 above)
Comment (thomasbnt) [reply public/locales/fr/common.json]: My bad :x 

Yes That'.s right, that's right! I wanted to correct this mistake for `récemment` but I PR without thinking about it again. For `recherchés` this also needs to be corrected. I still have no idea of the conjugation and time.

Comment (elamperti) [reply public/locales/fr/common.json]: Don't worry, the code reviews are perfect to iron out these details 😃

File: public/locales/fr/common.json (hunk 1 above)
Comment (elamperti) [reply public/locales/fr/common.json]: (Can you accept the suggested code change here? Thanks!)

**Pull request merged**